src/auth/types.ts

```diff
@@ -15,6 +15,7 @@ export type AzureAuthenticationSession = AuthenticationSession & {
 export type Tenant = {
     name: string;
     id: string;
+    countryCode?: string;
```

Comment:
@kejatura-dev I have started an internal discussion about this, I think we already have this data.